pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[tool.poetry]
+name = "coldtype"
+version = "0.5.9"
+readme = "README.md"
+description = "Functions for manual vectorized typesetting"
+authors = [
+  "Rob Stenson <rob.stenson@gmail.com>"
+]
+license = "Apache License 2.0"
+
+
+[tool.poetry.dependencies]
+python = "^3.7"
+lxml = "^4.6.3"
+fonttools = {extras = ["ufo"], version = "^4.25.1"}
+fontPens = "^0.2.4"
+more-itertools = "^8.8.0"
+easing-functions = "^1.0.4"
+timecode = "^1.3.1"
+mido = "^1.2.10"
+defcon = "^0.8.1"
+freetype-py = "^2.2.0"
+uharfbuzz = ">=0.14.0"
+unicodedata2 = "^13.0.0"
+python-bidi = "^0.4.2"
+rich = "^10.6.0"
+
+# Optional dependencies used by extras:
+skia-python = {version = ">=86.0", optional = true}
+glfw = {version = "^2.1.0", optional = true}
+PyOpenGL = {version = "^3.1.5", optional = true}
+PyOpenGL-accelerate = {version = "^3.1.5", optional = true}
+skia-pathops = {version = "^0.6.0", optional = true}
+SimpleWebSocketServer = {version = "^0.1.1", optional = true}
+watchdog = {version = "<2.0.0", optional = true}
+noise = {version = "^1.2.2", optional = true}
+ufo2ft = {version = "^2.23.0", optional = true}
+numpy = {version = "^1.21.1", optional = true}
+pynput = {version = "^1.7.3", optional = true}
+PyAudio = {version = "^0.2.11", optional = true}
+rtmidi = {version = "^2.3.4", optional = true}
+srt = {version = "^3.4.1", optional = true}
+
+
+[tool.poetry.extras]
+skia = ["skia-python"]
+viewer = [
+  "glfw",
+  "PyOpenGL",
+  "PyOpenGL-accelerate",
+  "skia-python",
+  "skia-pathops",
+  "SimpleWebSocketServer",
+  "watchdog",
+  "noise",
+  "ufo2ft",
+  "numpy"
+]
+c = ["srt", "noise"]
+webviewer = ["SimpleWebSocketServer", "watchdog"]
+experimental = ["pynput", "pyaudio", "rtmidi", "noise"]
+
+
+[tool.poetry.dev-dependencies]
+better-exceptions = "^0.3.3"
+pytest = "^6.2.4"
+
+
+[tool.poetry.scripts]
+coldtype = "coldtype.renderer:main"
```

Comment:
does the pyproject.toml file account for the package_data elements specified in the current setup.py? i.e. this part:

```python
include_package_data=True,
    package_data={
        "": [
            "webserver/webviewer.html",
            "demo/RecMono-CasualItalic.ttf",
            "demo/ColdtypeObviously-VF.ttf",
            "demo/MutatorSans.ttf",
            "demo/demo.py",
            "demo/blank.py",
            "demo/boiler.py",
            "renderer/picklejar.py",
        ],
    },
```

I guess more generally, is it just inferring what to include as packages (since they aren't being spelled out in something like the `packages=` line)

Reply:
By default, poetry will include everything in the `coldtype` directory (it uses the package name to infer what to include). If there's anything outside that which needs adding or removing, you can specify what to match with [include or exclude globs](https://python-poetry.org/docs/pyproject/#include-and-exclude).